apps/addons/management/commands/sign_addons.py

```diff
@@ -27,12 +29,12 @@ def handle(self, *args, **options):
                 'Please provide at least one addon id to sign. If you want to '
                 'sign them all, use the "process_addons --task sign_addons" '
                 'management command.')
-        signing_server = options.get('signing_server', settings.SIGNING_SERVER)
-        preliminary_signing_server = options.get(
-            'preliminary_signing_server', settings.PRELIMINARY_SIGNING_SERVER)
+        full_server = options.get('signing_server') or settings.SIGNING_SERVER
+        prelim_server = (options.get('preliminary_signing_server') or
+                         settings.PRELIMINARY_SIGNING_SERVER)
 
         addon_ids = [int(addon_id) for addon_id in args]
         with override_settings(
-                SIGNING_SERVER=signing_server,
-                PRELIMINARY_SIGNING_SERVER=preliminary_signing_server):
-            sign_addons(addon_ids)
+                SIGNING_SERVER=full_server,
+                PRELIMINARY_SIGNING_SERVER=prelim_server):
+            sign_addons(addon_ids, force=options['force'])
```

Comment:
Exactly, that's the reason. This management command is to be used with a "small" list of addon IDs. If you wish to sign all the addons, then you should use `./manage.py process-addons --task sign_addons`, which does use celery.

I could change it though, if you believe it would be more convenient. What do you think?